src/GithubSummary.js

```diff
@@ -9,10 +11,31 @@ import {
   ISSUE_EVENT, ISSUE_COMMENT_EVENT, PULL_REQUEST_EVENT, PULL_REQUEST_REVIEW_COMMENT_EVENT,
   CLOSED,
 } from './constants';
+import type { DefaultOptions, Options } from './types/Options';
+import type { Repo, User, Payload, Event } from './types/Github';
 
 export default class GithubSummary {
-  constructor(options) {
-    this.options = { ...GithubSummary.defaults, ...options };
+
+  static defaults: DefaultOptions = {
+    from: getStartOfDay(),
+    to: getEndOfDay(),
+    perPage: 100,
+    requestAllPages: false,
+    markdown: true,
+    formatter: '{checkbox} {avatar} <strong>{title}</strong>',
+    mergedTag: '<strong>merged</strong>',
+    closedTag: '<strong>closed</strong>',
+  };
+
+  static mergeOptions(a: any, b: any): Options {
```

Comment:
Had to add mergeOptions function since flow cannot do Object.assign with same types which as optional properties yet.